Civi/CCase/SequenceListener.php

```diff
@@ -64,7 +64,16 @@ public function onCaseChange(\Civi\CCase\Event\CaseChangeEvent $event) {
       }
     }
 
-    // OK, the activity has completed every step in the sequence!
+    //CRM-17452 - Close the case only if all the activities are complete
+    $activities = $analyzer->getActivities();
+    var_dump($activities);
```

Comment:
Can remove this line as we don't need to dump activities in $activities. 

(You can use CRM_Core_Error::debug_var('<variable name>', $activities); instead and you all get nicely formatted array in log. )